status/k8s.go

```diff
@@ -44,6 +44,9 @@ type K8sStatusParameters struct {
 
 	// The number of workers to use.
 	WorkerCount int
+
+	// The output format
+	Output string
```

Comment:
Would be nice to introduce a dedicated type for this field that will only take valid values `json` and `summary`.